nvme.c

```diff
@@ -4588,6 +4588,28 @@ static int ns_rescan(int argc, char **argv, struct command *cmd, struct plugin *
 	return err;
 }
 
+static int parse_sanact(char *str, __u8 *val)
+{
+	int len = strlen(str);
+
+	if (!strncasecmp(str, "exit-failure", len > 1 ? len : 1))
```

Comment:
Couldn't also accept the old accepted values here? E.g. Also the documentation should be updated accordingly

Reply:
By the change both the old accepted values and new string values are accepted. Noted I will update the documentation also.